extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -10,7 +10,33 @@
     <div class="fabric-canvas" ng-attr-id="<[$ctrl.canvasContainerId]>">
       <canvas ng-attr-id="<[$ctrl.canvasID]>" class="fabric-canvas-container"></canvas>
     </div>
-    <div ng-if="$ctrl.isDrawModePieChart()" class="fabric-canvas pie-chart-form">
+    <div ng-if="$ctrl.isDrawModeSVGUpload()" class="fabric-canvas custom-tool-form">
+      <div class="container upload-warning">
+        Note: The uploaded svg may not be rendered exactly.
+      </div>
+      <div ng-if="!$ctrl.isFileUploaded()">
+        <image-uploader on-file-changed="$ctrl.onFileChanged" allowed-image-type="$ctrl.imageType">
+        </image-uploader>
+      </div>
+      <div ng-if="$ctrl.isFileUploaded()">
+        <img ng-src="<[$ctrl.uploadedSVG]>" class="uploaded-svg-image">
+        <div class="container">
+          <label>
+            <input type="radio" ng-model="$ctrl.loadType" value="group" checked>
+            <span>
+              Load as Group
```

Comment:
Should I replace the labels as "Load each shape individually without grouping" and "Load after grouping the shapes" to provide a little more clarity and if so is it clear enough?

Reply:
I would suggest keeping the current label but add extra info on hover (eg. a tooltip) or some help text below explaining what the two options are. @aks681 do you have any thoughts on this?

Reply:
I have no idea what is going on, sorry. From a user perspective, what is the difference between the two options? If you could explain that clearly I could maybe try to advise on how to shorten it.

(In particular I do not understand what "grouping" means.)

Reply:
Based on previous discussion with @rohitkatlaa, there are two ways for the editor to treat a newly imported SVG, it can either interpret it as individual shapes that make up the image or treat it as an indivisible group. The former gives the user freedom to take apart the image or modify it freely (as if it were created entirely in the editor). The latter just allows resizing of the image. You can see these two scenarios demonstrated in the description of this PR (https://github.com/oppia/oppia/pull/10182#issue-464110698).

@rohitkatlaa did I get that right?

Reply:
Then perhaps: "Load as Single Image" vs "Load as Individual Shapes"?

Reply:
@kevintab95 Yes you got it correct. So should I go with @seanlip suggestion and replace the labels? And is more information required or it this clear enough?

Reply:
I have replaced the labels.